code/script.js

```diff
@@ -27,25 +39,196 @@ const showMessage = (message, sender) => {
           <p>${message}</p>
         </div>
       </section>
-    `
+    `;
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
-}
+  chat.scrollTop = chat.scrollHeight;
+};
 
-// Starts here
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage("Hi! Welcome. What's your name?", 'bot');
+};
+
+setTimeout(greetUser, 1000);
+
+const handleNameInput = (event) => {
+  console.log("event");
+  event.preventDefault();
+  userName = nameInput.value;
+  nameInput.value = "";
+  showMessage(`I'm ${userName}! Nice to meet you!`, 'user');
+  setTimeout(reply, 1000);
+};
+
+nameForm.addEventListener("submit", handleNameInput);
+
+function reply() {
+  console.log("reply");
+  showMessage(`Are you hungry, ${userName}? please choose your food.`, 'bot');
+
+
+
+  //This inject the buttons to the htlm
+  inputHandle.innerHTML = `
+<button class="send-btn" id="pizza">Pizza</button>
+<button class="send-btn" id="pasta">Pasta</button>
+<button class="send-btn" id="salad">Salad</button>
+
+`;
+
+  const foodButtons = inputHandle.querySelectorAll('.send-btn');
+  console.log("foodbuttons");
+  foodButtons.forEach(button => {
+    button.addEventListener('click', (event) => {
+      const selectedfood = event.target.id;
+      showMessage(`Amazing! You've selected ${selectedfood}!`, 'bot');
+      foodChoice(selectedfood);
+
+    });
+  });
 }
 
 // Set up your eventlisteners here
 
+function foodChoice(choice) {
+  console.log("foodchoice");
+  inputHandle.innerHTML = '';
+
+
+
+  if (choice === 'pizza') {
+    foodOptions = ['Napolitana', 'Margeritha', 'Bianca'];
+    foodMessage = 'Which pizza would you like to order? Choose from the menu.';
+  } else if (choice === 'pasta') {
+    foodOptions = ['Cacio e Pepe', 'Vongole', 'Carbonara'];
+    foodMessage = 'Which pasta would you like to order? Choose from the menu.';
+  } else if (choice === 'salad') {
+    foodOptions = ['Pomodoro e basilico', 'Buratta', 'Melon e Peperoncino'];
+    foodMessage = 'Which salad would you like to order? Choose from the menu.';
+  }
+
+  showMessage(foodMessage, 'bot');
+
+  // Dropdown menu
+
+  const dropdown = document.createElement('select');
+  console.log("select");
+  dropdown.id = 'food-dropdown';
+
+
+  dropdown.classList.add('dropdown-style');
+
+
+  foodOptions.forEach(option => {
+    console.log("option");
+    const optionElement = document.createElement('option');
+    optionElement.value = option;
+    optionElement.textContent = option;
+    dropdown.appendChild(optionElement);
+  });
+
```

Comment:
Do you mean that you want to check the option value here? 
```javascript
console.log(option) 
``` 
It is great to use the appendChild method here. 👍